content/blog/2017-11-11-rperseus.md

```diff
@@ -0,0 +1,106 @@
+
+![](/img/blog-images/2017-11-17-rperseus/nantg.png)
+*1 Corinithians 13 in Nestle-Aland Novum Testamentum Graece*
+
+Ignoring that some scholars exclude Colossians from the “authentic” letters, let’s see the references alongside each other:
+
+```r
+library(rperseus) #devtools::install_github(“ropensci/rperseus”)
+library(tidyverse)
+
+tribble(
+  ~label, ~excerpt,
+  "Colossians", "1.4",
+  "1 Thessalonians", "1.3",
+  "1 Thessalonians", "5.8",
+  "Romans", "8.35-8.39"
+  ) %>% 
+  left_join(perseus_catalog) %>%
+  filter(language == "eng") %>%
+  select(urn, excerpt) %>%
+  pmap_df(get_perseus_text) %>%
+  perseus_parallel(words_per_row = 4)
+```
+![](/img/blog-images/2017-11-17-rperseus/Parallel1.png)
+
+A brief explanation: First, I specify the labels and excerpts within a tibble. Second, I join the lazily loaded `perseus_catalog` onto the data frame. Third, I filter for the Greek[^2] and select the columns containing the arguments required for `get_perseus_text`. Fourth, I map over each urn and excerpt, returning another data frame. Finally, I pipe the output into `perseus_parallel`.
+
+[^2]: The Greek text from the Perseus Digital Library is from 1885 standards. The advancement of textual criticism in the 20th century led to a more stable text you would find in current editions of the Greek New Testament.  
+
+The key word shared by each passage is *agape* (“love”). Without going into details, it might be fruitful to consider the references alongside each other, perhaps pondering how the semantic range of *agape* expands or contracts within the Pauline corpus. 
+
+If your Greek is rusty, you can flip the `language` filter to “eng” to view an older English translation.[^3] And if the margin references a text from the Old Testament, you can call the Septuagint as well as the original Hebrew: 
+
+```r
+tribble(
+  ~label, ~excerpt,
+  "Genesis", "32.31",
+  "Genesis, pointed", "32.31",
+  "Numeri", "12.8",
+  "Numbers, pointed", "12.8"
+  ) %>% 
+  left_join(perseus_catalog) %>%
+  filter(language %in% c("grc", "hpt")) %>%
+  select(urn, excerpt) %>%
+  pmap_df(get_perseus_text) %>%
+  perseus_parallel()
+```
+![](/img/blog-images/2017-11-17-rperseus/Parallel2.png)
+
+[^3]: The English translation is from Rainbow Missions, Inc. *World English Bible.* Rainbow Missions, Inc.; revision of the American Standard Version of 1901. I’ve toyed with the idea of incorporating more modern translations, but that would require require resources beyond the Perseus Digital Library. 
+
+Admittedly, there is some “friction” here in joining the `perseus_catalog` onto the initial tibble. There is a learning curve with getting acquainted with the idiosyncrasies of the catalog object. A later release will aim to streamline this workflow. 
+
+Hundreds of other texts are available through rperseus, including Plato, Aristotle, Virgil, Cicero, the Boethius. 
```

Comment:
This line might be helpful closer to the top of the post so people can more clearly see the breadth of applications of `rperseus`. Seems to go best with this line: "The library includes a catalog of classical texts, a Greek and Latin lexicon, and a word study tool for appearances and references in other literature." since audience will relate to accessing Plato texts more than saying only Greek, Latin etc.

That way, your personal-passion example of biblical texts sounds less niche as reader can better visualize other examples.

Reply:
Good call, I moved it up.